config/managers.ini.template

```diff
@@ -2,6 +2,8 @@
 [manifest]
 ;domain = @MANIFEST_SERVICE_HOSTNAME@
 ;domainPort = 80
+[manifest][rendition]
```

Comment:
This is not how sub-sections should be written in ini file.
Use: `rendition.maxRetries = 6` instead.